array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
+function dropRight(array, count) {
+  if (array.length === 0) {
+    return [];
+  } else {
+    var number = array.length - count;
+    array.length = number;
+    return array;
+  }
```

Comment:
This mutates the original array and accomplishes the task in an incorrect way.  You want to use a loop and do you best to not modify the original array.